style.scss

```diff
@@ -3431,5 +3627,4 @@ body >footer {
       padding: 25px 0 20px 0;
     }
   }
-}
-
+}
```

Comment:
I like them because it's typical for all lines to end with `\n` with most UNIX text editors, and because github shows a weird icon if the line doesn't end with a newline!